chrome_manifest_v3/views/popupView.js

```
@@ -1406,7 +1421,7 @@ Gmail2Trello.PopupView.prototype.updateCards = function (tempId = 0) {
 
 // Select/de-select attachments and images based on first button's state:
 Gmail2Trello.PopupView.prototype.toggleCheckboxes = function (tag) {
-  var $jTags = $('#' + tag + ' input[type="checkbox"]', self.$popup);
+  let $jTags = $('#' + tag + ' input[type="checkbox"]', this.$popup);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)